src/views/Farms/components/FarmTable/Apr.tsx

```diff
@@ -37,7 +38,12 @@ const Container = styled.div`
 
 const AprWrapper = styled.div`
   min-width: 60px;
-  text-align: left;
+  text-align: right;
+  margin-right: 14px;
```

Comment:
Margins should be in multiples of 4px.

Reply:
Ah I think this is a crab cake introduction  - inherited from them. I'll fix it up in the other places it exists in farms too.